lib/cryptoexchange/helpers/hash_helper.rb

```diff
@@ -0,0 +1,20 @@
+module HashHelper
+  class << self
+    # "Polyfill" of the Hash#dig method from Ruby 2.3
+    #
+    #   Examples
+    #
+    #     > dig({a: {b: 1}}, :a, :b)
+    #     => 2
+    #
+    #     > dig({a: {b: 1}}, :a, :c)
+    #     => nil
+    def dig(obj, *args)
```

Comment:
@dominicwong617 @tmlee I'm on the fence for this one. On one hand, I think we should keep Ruby 2.2 because as a library, we should be as flexible as we reasonably can be, especially in the context of external dependencies (Ruby included). In this case, I think simple polyfills like this are reasonable. On the other hand, Ruby 2.2 is actually quite old now and Ruby core [is not supporting it][1] anymore.

[1]: https://www.ruby-lang.org/en/news/2017/03/28/ruby-2-2-7-released/